coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/loader/sql/AbstractSqlLogger.scala

```diff
@@ -175,7 +175,7 @@ abstract class AbstractSqlLogger[CT, CV](val connection: Connection,
 
   def secondaryAddIndex(fieldName: ColumnName) = {
     checkTxn()
-    logLine(SecondaryAddIndex, messages.SecondaryAddIndex.defaultInstance)
+    logLine(SecondaryAddIndex, messages.SecondaryAddIndex(fieldName.caseFolded))
```

Comment:
This one _should_ be `name`, not `caseFolded` because it'll get put back into a `ColumnName` on the other end